orchagent/portsorch.cpp

```diff
@@ -2484,6 +2484,27 @@ bool PortsOrch::getQueueTypeAndIndex(sai_object_id_t queue_id, string &type, uin
     return true;
 }
 
+bool PortsOrch::isAutoNegEnabled(sai_object_id_t id)
```

Comment:
Why do we  use m_autoneg in port to identify if autoneg is enabled or not?

Reply:
This is in the context of `initializePort`. I believe at that time, `m_autoneg` is not initialized.